src/utils/middleware.ts

```diff
@@ -23,6 +24,47 @@ interface IServiceProviderOrganization {
   displayName: string;
   name: string;
 }
+
+export enum ContactType {
+  OTHER = "other"
+}
+
+export enum EntityType {
+  AGGREGATOR = "spid:aggregator"
+}
+
+export enum AggregatorType {
+  PublicServicesFullOperator = "PublicServicesFullOperator"
+}
+
+const CommonExtension = t.partial({
+  FiscalCode: t.string,
+  IPACode: t.string,
+  VATNumber: t.string
+});
```

Comment:
This fileds are all required for a `Gestore`
```suggestion
const CommonExtension = t.interface({
  FiscalCode: t.string,
  IPACode: t.string,
  VATNumber: t.string
});
```